### PR TITLE
Add OHBM Aperture citation

### PR DESCRIPTION
**Pull request by gdoubleyew**

### CODE REVIEW

File: _data/pubs.yml [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+      - Xia Zhu
+      - Mihai Capota
+      - Jonathan D. Cohen
+      - Uri Hasson
+      - Kai Li
+      - Peter J. Ramadge
+      - Nicholas B. Turk-Browne
+      - Theodore L. Willke
+      - Kenneth A. Norman
+  journal: OHBM Aperture
```
Comment (manojneuro) [inline _data/pubs.yml:41]: Please change "OHBM Aperture" to preprint.

File: tutorials/index.md [hunk 2]
```diff
@@ -420,6 +420,8 @@ We have compiled a list of useful resources that cover the basics of GitHUB, Pyt
 
 Like BrainIAK itself, these tutorials are open-sourced as a service to the field and can be freely used, modified, and shared. They are licensed under the Apache License, Version 2.0. If you find them helpful in your research or re-use code they contain, you might consider citing:
 
+Manoj Kumar, Michael J. Anderson, James W. Antony, Christopher Baldassano, Paula P. Brooks, Ming Bo Cai, Po-Hsuan Cameron Chen, Cameron T. Ellis, Gregory Henselman-Petrusek, David Huberdeau, J. Benjamin Hutchinson, Peeta Y. Li, Qihong Lu, Jeremy R. Manning, Anne C. Mennen, Samuel A. Nastase, Hugo Richard, Anna C. Schapiro, Nicolas W. Schuck, Michael Shvartsman, Narayanan Sundaram, Daniel Suo, Javier S. Turek, Vy A. Vo, Grant Wallace, Yida Wang, Hejia Zhang, Xia Zhu, Mihai Capota, Jonathan D. Cohen, Uri Hasson, Kai Li, Peter J. Ramadge, Nicholas B. Turk-Browne, Theodore L. Willke, Kenneth A. Norman, (2020, December 9). BrainIAK: The Brain Imaging Analysis Kit. OHBM Aperture, [https://doi.org/10.31219/osf.io/db2ev](https://doi.org/10.31219/osf.io/db2ev)
+
```
Comment (manojneuro) [inline tutorials/index.md:424]: Please change "OHBM Aperture" to preprint.

File: _data/pubs.yml [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+      - James W. Antony
+      - Christopher Baldassano
+      - Paula P. Brooks
+      - Ming Bo Cai
+      - Po-Hsuan Cameron Chen
+      - Cameron T. Ellis
+      - Gregory Henselman-Petrusek
+      - David Huberdeau
+      - J. Benjamin Hutchinson
+      - Peeta Y. Li
```
Comment (manojneuro) [inline _data/pubs.yml:16]: Please make this "Y. Peeta Li" (per author request). instead of Peeta Y. Li

**Comment by gdoubleyew** [comment]
Here are the new screen shots
[cid:image001.png@01D6CFB4.12995570]
[cid:image002.png@01D6CFB4.12995570]

File: tutorials/index.md (hunk 2 above)
Comment (manojneuro) [reply tutorials/index.md:424]: For consistency with the other citation, I have modified a few things (removed italics, and the month) below. Please use this:

Manoj Kumar, Michael J. Anderson, James W. Antony, Christopher Baldassano, Paula P. Brooks, Ming Bo Cai, Po-Hsuan Cameron Chen, Cameron T. Ellis, Gregory Henselman-Petrusek, David Huberdeau, J. Benjamin Hutchinson, Y. Peeta Li, Qihong Lu, Jeremy R. Manning, Anne C. Mennen, Samuel A. Nastase, Hugo Richard, Anna C. Schapiro, Nicolas W. Schuck, Michael Shvartsman, Narayanan Sundaram, Daniel Suo, Javier S. Turek, Vy A. Vo, Grant Wallace, Yida Wang, Hejia Zhang, Xia Zhu, Mihai Capota, Jonathan D. Cohen, Uri Hasson, Kai Li, Peter J. Ramadge, Nicholas B. Turk-Browne, Theodore L. Willke, Kenneth A. Norman, 2020. BrainIAK: The Brain Imaging Analysis Kit. preprint, [https://doi.org/10.31219/osf.io/db2ev](https://doi.org/10.31219/osf.io/db2ev)

File: _data/pubs.yml [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+      - James W. Antony
+      - Christopher Baldassano
+      - Paula P. Brooks
+      - Ming Bo Cai
+      - Po-Hsuan Cameron Chen
+      - Cameron T. Ellis
+      - Gregory Henselman-Petrusek
+      - David Huberdeau
+      - J. Benjamin Hutchinson
+      - Y. Peeta Li
```
Comment (manojneuro) [inline _data/pubs.yml:16]: LGTM.

File: _data/pubs.yml [hunk 5]
```diff
@@ -1,3 +1,50 @@
...
+      - Xia Zhu
+      - Mihai Capota
+      - Jonathan D. Cohen
+      - Uri Hasson
+      - Kai Li
+      - Peter J. Ramadge
+      - Nicholas B. Turk-Browne
+      - Theodore L. Willke
+      - Kenneth A. Norman
+  journal: preprint
```
Comment (manojneuro) [inline _data/pubs.yml:41]: LGTM

File: tutorials/index.md (hunk 2 above)
Comment (manojneuro) [reply tutorials/index.md:424]: LGTM.

File: tutorials/index.md [hunk 6]
```diff
@@ -420,6 +420,8 @@ We have compiled a list of useful resources that cover the basics of GitHUB, Pyt
...
 
+Manoj Kumar, Michael J. Anderson, James W. Antony, Christopher Baldassano, Paula P. Brooks, Ming Bo Cai, Po-Hsuan Cameron Chen, Cameron T. Ellis, Gregory Henselman-Petrusek, David Huberdeau, J. Benjamin Hutchinson, Y. Peeta Li, Qihong Lu, Jeremy R. Manning, Anne C. Mennen, Samuel A. Nastase, Hugo Richard, Anna C. Schapiro, Nicolas W. Schuck, Michael Shvartsman, Narayanan Sundaram, Daniel Suo, Javier S. Turek, Vy A. Vo, Grant Wallace, Yida Wang, Hejia Zhang, Xia Zhu, Mihai Capota, Jonathan D. Cohen, Uri Hasson, Kai Li, Peter J. Ramadge, Nicholas B. Turk-Browne, Theodore L. Willke, Kenneth A. Norman, 2020. BrainIAK: The Brain Imaging Analysis Kit. preprint, [https://doi.org/10.31219/osf.io/db2ev](https://doi.org/10.31219/osf.io/db2ev)
+
 Kumar, M., Ellis, C.T., Lu, Q., Zhang, H., Capotă, M., Willke, T.L., Ramadge, P.J., Turk-Browne, N.B. and Norman, K.A., 2020. BrainIAK tutorials: User-friendly learning materials for advanced fMRI analysis. PLoS Computational Biology, 16(1), e1007549. [https://doi.org/10.1371/journal.pcbi.1007549](https://doi.org/10.1371/journal.pcbi.1007549)
 
 </div>
```
Comment (manojneuro) [inline tutorials/index.md:427]: @gdoubleyew I just realized that there is one more file that hasn't been updated with the citations: https://brainiak.org/docs/index.html?highlight=citing.

**Review by manojneuro — CHANGES_REQUESTED** [review]

**Review by manojneuro — APPROVED** [review]
LGTM

**Pull request merged**